save-frontend/src/main/kotlin/com/saveourtool/save/frontend/externals/i18next/InitI18n.kt

```diff
@@ -0,0 +1,41 @@
+/**
+ * File containing initialization of i18next
+ */
+
+package com.saveourtool.save.frontend.externals.i18next
+
+/**
+ * Function that encapsulates i18n initialization.
+ *
+ * @param isDebug flag to set debug mode
+ * @param interpolationEscapeValue interpolation.escapeValue value
+ */
+@Suppress("UNUSED_PARAMETER")
+fun initI18n(isDebug: Boolean = false, interpolationEscapeValue: Boolean = false) {
+    js("""
+        var i18n = require("i18next");
+        var reactI18n = require("react-i18next");
+        var resources = {
+            en: {
+                translation: {
+                    "Notifications": "Notifications"
```

Comment:
Then there will be a json reading?

Reply:
Yep, there's an ability to pass a path to file